editor/src/clj/editor/pipeline/texture_set_gen.clj

```diff
@@ -77,38 +76,73 @@
     :sprite-trim-mode-7 7
     :sprite-trim-mode-8 8))
 
+(defn- texture-set-layout-rect
+  ^TextureSetLayout$Rect [{:keys [path width height]}]
+  (let [id (resource/proj-path path)]
+    (TextureSetLayout$Rect. id -1 (int width) (int height))))
+
+(defonce ^:private ^TextureSetProto$SpriteGeometry rect-sprite-geometry-template
+  (let [points (vector-of :float -0.5 -0.5 -0.5 0.5 0.5 0.5 0.5 -0.5)
+        uvs (vector-of :float 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0)
```

Comment:
Looking at `TextureSetGenerator/buildConvexHull`, which is being used when Sprite Trimming is enabled, it also writes `0.0` for all UV coordinates. I think this is something that could be used by the engine in the future, but currently isn't, perhaps.